docs/pages/identity-governance/access-requests/automatic-reviews.mdx

```diff
@@ -26,48 +27,28 @@ requests access to the `access` role.
 
 - This feature requires Teleport Identity Governance.
 
-## Step 1/4. Create a requester role and user
```

Comment:
Note that the prerequisites for this page only say Teleport 18.0.0 or higher, but the resource-based reviews will require 18.2.0 or higher. Can we clarify?

Reply:
Good call. I've bumped the required version.